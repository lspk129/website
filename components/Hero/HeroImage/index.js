import styled from 'styled-components';

const HomeImage = styled.div`
  background-image: url("../../../static/img21.jpg");
  background-repeat: no-repeat;
  background-attachment: fixed;
  background-size: cover;
  display: grid;
  grid-template-rows: 1fr 5fr;
  align-items: start;
  justify-items: center;
`;

export default HomeImage;
