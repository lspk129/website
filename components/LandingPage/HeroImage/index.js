import styled from 'styled-components';

export default styled.div`
  background-image: url("http://trendytheme.net/demo2/wp/69/multipage/wp-content/uploads/revslider/web-product-dark/webproduct_darkbg2.jpg");
  background-position: center center;
  background-repeat: no-repeat;
  background-attachment: fixed;
  background-size: cover;
  margin: -10px;
  height: 100vh
`;

