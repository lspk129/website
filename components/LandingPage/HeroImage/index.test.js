import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';


import HeroImage from './index';

describe('HeroImage component', () => {
  it('renders image', () => {
    const component = renderer.create(<HeroImage />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe('HeroImage component', () => {
  it('renders div with class', () => {
    const component = shallow(<HeroImage />);
    expect(component.find('div').length).toEqual(1);
  });
});
