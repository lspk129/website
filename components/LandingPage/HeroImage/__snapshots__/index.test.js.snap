// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroImage component renders image 1`] = `
<div
  className="kHXxIQ"
/>
`;
